### PR TITLE
Invert trace images in dark mode

### PR DESCRIPTION
**Pull request by AntonKhorev**
Do 85% invert because the page background is not 100% black.

image
image

### CODE REVIEW

File: app/assets/stylesheets/common.scss [hunk 1]
```diff
@@ -1036,4 +1036,12 @@ div.secondary-actions {
   margin-bottom: -1px;
 }
 
+/* Rules for traces */
+
+@include color-mode(dark) {
+  img.trace_image {
```
Comment (tordans) [inline app/assets/stylesheets/common.scss:1042]: Should this maybe be a custom utility class? The utility class name would tell more about what it does. Something like 'dark-invert-image'.

Comment (HolgerJeromin) [reply app/assets/stylesheets/common.scss:1042]: this name would be wrong is non-dark style.
Or do I miss something?

Comment (tordans) [reply app/assets/stylesheets/common.scss:1042]: My point is not the suggested name but changing the name from a specific element name to a utility class name.

**Comment by tomhughes** [comment]
Merged #4681 in preference to this.

**Pull request closed without merging**